src/components/MainNavbar.js

```diff
@@ -21,6 +21,10 @@ class MainNavbar extends React.Component {
             <NavLink className="header-text">Open Projects</NavLink>
           </LinkContainer>
 
+          <LinkContainer to="/faq">
+            <NavLink className="header-text">FAQ</NavLink>
+          </LinkContainer>
+
```

Comment:
small thing - thoughts on having the FAQ as the last item in the nav?

Reply:
I'm a fan of this idea!